drivers/sensor/ite_vcmp_it8xxx2/Kconfig

```diff
@@ -12,3 +12,31 @@ config VCMP_IT8XXX2
 	  This option enables the ITE it8xxx2 voltage comparator,
 	  it8xxx2 supports six 10-bit resolution voltage comparator
 	  channels, and the input of each comparator comes from ADC pin.
+
+if VCMP_IT8XXX2
+
+config VCMP_IT8XXX2_WORKQUEUE
+	bool "ITE it8xxx2 voltage comparator threshold detection uses internal work queue"
+	help
+	  Threshold detection ISR utilizes system work queue for calling
+	  trigger handlers; set this option to use dedicated work queue instead.
+
+if VCMP_IT8XXX2_WORKQUEUE
+
+config VCMP_IT8XXX2_WORKQUEUE_PRIORITY
+	int "ITE it8xxx2 voltage comparator threshold detection work queue priority"
+	default SYSTEM_WORKQUEUE_PRIORITY
+	help
+	  This option sets internal ITE voltage comparator threshold detection
+	  workqueue priority.
+
+config VCMP_IT8XXX2_WORKQUEUE_STACK_SIZE
+	int "ITE it8xxx2 voltage comparator threshold detection work queue stack size"
+	default 768
+	help
+	  This option sets internal ITE voltage comparator threshold detection
+	  workqueue stack size.
+
```

Comment:
Please add a definition for IT8XXX2_VCMP_INIT_PRIORITY, including prompt and help text.

Reply:
Done